planner/core/exhaust_physical_plans.go

```diff
@@ -684,7 +684,12 @@ func (p *LogicalJoin) constructInnerTableScanTask(
 	for i := range ds.stats.Cardinality {
 		ds.stats.Cardinality[i] = 1
 	}
-	rowSize := ds.TblColHists.GetTableAvgRowSize(ds.TblCols, ts.StoreType, true)
+	var rowSize float64
+	if p.ctx.GetSessionVars().EnableChunkRPC {
+		rowSize = ds.TblColHists.GetTableAvgRowSizeChunkFormat(ds.TblCols, ts.StoreType, true)
+	} else {
+		rowSize = ds.TblColHists.GetTableAvgRowSize(ds.TblCols, ts.StoreType, true)
```

Comment:
Can we check `EnableChunkRPC` in functions like `GetTableAvgRowSize`, then we don't need to add this check everywhere?

Reply:
Addressed.